### DISCUSSION TITLE
Migrate Profile from entra hybrid joined computer to entra ad joined computer

### DISCUSSION THREAD

**Discussion by gonpozapalopo** [belowaverage-org/SuperGrate#116] [Q&A] [answered]
Hi, first of all congratulations for the hard work.
I would like to know if is it possible to migrate a profile from an entra hybrid joined computer to an entra ad joined computer. I have been using usmt for years and had no problem migrating profiles. But when I run the usmt loadstate on an entra ad joined computer from a previous scanstate run  on an entra ad joined computer it will succeed to perform the backup , but when the user logs in it creates a new profile. I believe supergrate relies on usmt, so I was wondering if this scenario is covered by supergrate.  I am going to try it and put the results here, but I was wondering if someone had the same trouble and if the tool coverd this use case. Thanks again for the effort and your work

**Comment by krisdb2009** [comment, marked as answer]
Hi!
You should be able to accomplish this with this feature:
https://belowaverage.org/documentation/supergrate/#:~:text=Press%20%E2%80%9CStart%E2%80%9D.-,Rename%20User%20(Optional),Click%20%E2%80%9COK.%E2%80%9D,-Performing%20a%20Full

Let me know if that works!

**Comment by gonpozapalopo** [comment]
Thanks for the reply, yes I have already tried it successfully.
Thanks a lot for the support , great job and a great tool

**Reply by krisdb2009** [reply]
Happy to hear that worked!

**Comment by gonpozapalopo** [comment]
I have tried this scenario and as I told you it is working fine. I have installed a personal digital certificate to see if with the default settings it got migrated, and it didn´t.

I would like to ask if we can migrate personal digital certificates with supergrate, and if positive, what configuration is needed. Thanks in advance for your help and support.
